src/components/app/details/main.tsx

```diff
@@ -88,65 +88,88 @@ export default function AppDetailsPage({ isV2 }: AppDetailsProps) {
-        setGroupFilterOptions([])
-        const { result } = await getEnvGroupList(+appId, FilterParentType.app)
-        if (result) {
-            const _groupFilterOption = []
-            let _selectedGroup
+        setSelectedAppList([]);
+        setAppListLoading(true);
+        setGroupFilterOptions([]);
+      
+        try {
+          const { result } = await getEnvGroupList(+appId, FilterParentType.app);
+      
+          if (result) {
+            const _groupFilterOption = [];
+            let _selectedGroup;
+      
             for (const group of result) {
-                const processedGroupData = {
-                    value: group.id.toString(),
-                    label: group.name,
-                    // @ts-ignore
-                    appIds: group.resourceIds,
-                    description: group.description,
-                }
-                _groupFilterOption.push(processedGroupData)
-                if (groupId && groupId === group.id) {
-                    _selectedGroup = processedGroupData
-                }
+              const processedGroupData = {
+                value: group.id.toString(),
+                label: group.name,
+                // @ts-ignore
+                appIds: group.resourceIds,
+                description: group.description,
+              };
+      
+              _groupFilterOption.push(processedGroupData);
+      
+              if (groupId && groupId === group.id) {
+                _selectedGroup = processedGroupData;
+              }
             }
+      
             if (_selectedGroup) {
-                const selectedAppsMap: Record<string, boolean> = {}
-                const groupAppIds = _selectedGroup?.appIds || []
-                for (const appId of groupAppIds) {
-                    selectedAppsMap[appId] = true
-                }
-                setSelectedAppList(appListOptions.filter((app) => selectedAppsMap[app.value]))
-                setSelectedGroupFilter([_selectedGroup])
+              const selectedAppsMap: Record<string, boolean> = {};
+              const groupAppIds = _selectedGroup?.appIds || [];
+      
+              for (const appId of groupAppIds) {
+                selectedAppsMap[appId] = true;
+              }
+      
+              setSelectedAppList(appListOptions.filter((app) => selectedAppsMap[app.value]));
+              setSelectedGroupFilter([_selectedGroup]);
             } else {
-                setSelectedAppList([])
-                setSelectedGroupFilter([])
+              setSelectedAppList([]);
+              setSelectedGroupFilter([]);
             }
-            _groupFilterOption.sort(sortOptionsByLabel)
-            setGroupFilterOptions(_groupFilterOption)
+      
+            _groupFilterOption.sort(sortOptionsByLabel);
+            setGroupFilterOptions(_groupFilterOption);
+          }
+        } catch (error) {
+          console.error('Error fetching environment group list:', error);
```

Comment:
We can import showError and use it to show error in toast by calling showError(error)